.github/workflows/ci.ubuntu-x64.dev.yml

```
@@ -0,0 +1,54 @@
+  push:
+    branches: [main, dev]
+  pull_request:
+    branches: [main, dev]
+
+jobs:
+  docker-ci:
+    runs-on: ubuntu-latest
+    name: Docker CI (ubuntu-x64.dev)
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)